.github/workflows/monthly.yml

```diff
@@ -0,0 +1,41 @@
+name: monthly
+
+on:
+  push
+    branches:
+      - master
+
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  run:
+    name: Refresh month of man pages
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler: none
+
+      - name: Install dependencies
+        run: rake setup
+
+      - name: Update month
+        run: bin/rake man:build
+
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v4
+        with:
+          author_name: Utkarsh Gupta
+          author_email: utkarsh@debian.org
+          message: "Bump man pages month"
+          add: "bundler/man/*"
```

Comment:
Another way of doing this is:

```
      - name: Set up git config
        run: |
          git config user.name "Utkarsh Gupta"
          git config user.email "utkarsh@debian.org"
      - name: Commit and push the changes
        run: |
          git add bundler/man/*
          git commit -m "Bump man pages month"
          git push origin master
```

Reply:
Mmmm, don't really have a strong opinion, I guess since it's so simple we might do it ourselves instead of relying on an external action?

Reply:
Sure, I'll change it to this other way I mentioned above^ :smile: